README.md

```diff
@@ -202,6 +202,16 @@ Documentation for using each command is available by typing:
 > python manage.py help <command>
 ```
 
+### Pickle Tribe gene sets
+
+This step downloads and pickles all the public Tribe gene sets for every
+organism that has been loaded into the database. These pickled files are
+necessary for doing gene set enrichment analyses.
```

Comment:
Oh also @mhuyck one more question - since I bumped the tribe-client version number, I should probably update the requirements.txt for adage and include it here, right? Even though the only thing that will have changed in tribe-client is the README, but we should keep them sync'ed anyway?